app/components/NewsList/NewsList.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { News } from '@/models/types';
+import styles from './NewsList.module.scss';
+import Article from '../Article/Article';
+
+type Props = {
+  news: News[];
+};
+
+function NewsList({ news }: Props) {
+  return (
+    <div className={styles.newsList}>
+      {news.map((article) => (
+        <Article article={article} key={article.url} />
```

Comment:
Please install uuid package and inside getNews when you get data use map to generate ids so you’ll be able to use it for key

Reply:
It doesn’t make much sense to generate keys directly in key, because on each render all old elements will generate new keys and will be consider by react as new things even though they are the same. You can use index and you’ll get the same result. So it’s better generate ids in getNews when you request them from an API